### ISSUE TITLE
Thanks, but the  results on Pre@1 of something-something is too bad, could you show your results and parameter configuration exactly？

### ISSUE THREAD

**Issue by nemonameless** [coderSkyChen/Action_Recognition_Zoo#2]
Due to the challenge of Something-something dataset and the limitations of our GPUs  ? 
could you show your results and parameter configuration exactly？

**Comment by coderSkyChen** [comment]
Thanks, after carefully review, i've found that the input of the torch.nn.CrossEntropyLoss() should be the logits and ground truth, however in the previous version, the softmax function is used in forward pass, making the training results bad, i've debugged.
